Sources/GRPCInteropTests/Generated/empty_service.grpc.swift

```diff
@@ -86,7 +86,7 @@ extension Grpc_Testing_EmptyService.ClientProtocol {
 @available(macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0, *)
 public struct Grpc_Testing_EmptyService_Client: Grpc_Testing_EmptyService.ClientProtocol {
     private let client: GRPCCore.GRPCClient
-    
```

Comment:
Did the formatter complain about / fix this? It should be ignored from formatting because it's generated. Same goes for all "*.pb.swift" and "*.grpc.swift".